iot/api/CloudIotSample/Program.cs

```diff
@@ -1209,6 +1236,142 @@ public static object UnbindDeviceFromGateway(string projectId, string cloudRegio
         }
         //[END iot_list_devices_for_gateway]
 
+        //[START iot_unbind_all_devices]
+        public static object UnbindAllDevices(string projectId, string cloudRegion, string registryId)
+        {
+            var cloudIot = CreateAuthorizedClient();
+            // The resource name of the location associated with the key rings.
+            var parent = $"projects/{projectId}/locations/{cloudRegion}/registries/{registryId}";
+            try
+            {
+                var devices = cloudIot.Projects.Locations.Registries.Devices.List(parent).Execute().Devices;
+                Console.WriteLine("Devices: {0}", parent);
+
+                if (devices != null)
+                {
+                    devices.ToList().ForEach(response =>
+                    {
+                        UnbindDeviceFromAllGateways(cloudIot, projectId, cloudRegion, registryId, response.Id);
```

Comment:
All lines of code between START and END tags should be <= 80 characters long.  How many characters in this line?

Reply:
Okay, I will shorten this.